org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "http://schemas.opentranslatorstothings.org/org.opent2t.sample.hub.superpopular#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "",
+  "title": "OpenT2T Hub",
+  "definitions": {
+    "org.opent2t.sample.hub.superpopular": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "ReadOnly, Gets the name of the hub."
+        },
+        "devices": {
+          "description": "ReadOnly, Gets the current set of devices associated with the hub."
+        }
```

Comment:
Regarding support for both OCF and AllJoyn, couldn't we differentiate based on how we're calling into the translator?

We interact with OCF using getHubResURI, and we interact with AllJoyn using getDevices.

getHubResURI could return a list of OCF devices, while getDevices can return whatever AllJoyn would expect to get back, at least from the enumerate side, I suppose translator initialization would be impact in that case though.

This is probably fine for now, but I think we should at least think about something more efficient in the future.

---
In reply to: [76344264](https://github.com/openT2T/translators/pull/75#discussion_r76344264) [](ancestors = 76344264,76336787,76335569,76334693,76154047)